spec/serializers/important_events_serializer_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ImportantEventsSerializer do
+  subject(:serializer) { described_class.new(event) }
+
+  let(:event) { create(:event_move_approve) }
```

Comment:
maybe we should use an important event here instead (either incident or medical) to show that the classification will come back as either of those?

Reply:
I don't think it makes much difference to be honest; the classification behaviour already has coverage at unit level. But happy to change either way.

Reply:
Updated